[2024-08] Content and structure update (#11)
justaugustus/about: content/_index.md

tagline = "Caesar of Systems"
+++

> Stephen is a Black engineering director and leader in open source communities, based in New York City.

Reviewer: how about: Stephen is a Black engineering director and community leader in open source, based in New York City.

Author: ```suggestion
Stephen is a Black engineering director and community leader in open source, based in New York City.
```

He is the Head of [Open Source at Cisco][cisco-open-source], working within the
Corporate Strategy, Development, and Incubation (CSDI) organization.

> Outside of managing Cisco's open source strategy and a team of brilliant architects in Cisco's Open Source Program Office (OSPO), Stephen participates in foundation-level governing boards (OpenSSF, CNCF, OpenAPI Initiative, project-level steering committees (Kubernetes, OpenSSF Scorecard, TODO Group), and helps maintain a few codebases around open source projects you may have heard of.

Reviewer: missing a closing bracket after OpenAPI Initiative

Author: ```suggestion
Outside of managing Cisco's open source strategy and a team of brilliant architects in Cisco's Open Source Program Office (OSPO), Stephen participates in foundation-level governing boards (OpenSSF, CNCF, OpenAPI Initiative), project-level steering committees (Kubernetes, OpenSSF Scorecard, TODO Group), and helps maintain a few codebases around open source projects you may have heard of.
```

> Additionally, he serves as an advisor and invests in startups with open source in their DNA.

Reviewer: how about: Additionally, he serves as a startup advisor and avid investor in young companies with open source in their DNA.

Author: Yeah, I'm going to play with this one a little.

Author: Fixed in https://github.com/justaugustus/about/pull/11/commits/ccb73c31273a02c7d5f68108ae49be8dd85b47a7.

> Within the Open Source Security Foundation, he maintains OpenSSF Scorecard, a project that can help you assess the security posture of open source projects.

Reviewer: ```suggestion
Within the Open Source Security Foundation (OpenSSF), he maintains OpenSSF Scorecard, a project that helps assess the security posture of open source projects.
```

> For [Kubernetes][kubernetes], Stephen serves as a [Steering Committee][k8s-steering] member and a Chair for [SIG Release][sig-release].

Reviewer: ```suggestion
For [Kubernetes][kubernetes], Stephen serves as a [Steering Committee][k8s-steering] member and is the Chair of [SIG Release][sig-release].
```

Author: ```suggestion
For [Kubernetes][kubernetes], Stephen serves as a [Steering Committee][k8s-steering] member and a chair of [SIG Release][sig-release].
```

3 chairs total

> He has co-founded transformational elements of the project, including the [KEP][enhancements] (Kubernetes Enhancements Proposal) process, the [Release Engineering][releng] subproject, and Working Group Naming. He has also previously served as a chair for both SIG PM and SIG Azure.

Reviewer: ```suggestion
He has co-founded transformational elements of the project, including the [KEP][enhancements] (Kubernetes Enhancements Proposal) process, the [Release Engineering][releng] subproject, and Working Group Naming. He has previously served as Chair of both SIG PM and SIG Azure.
```

Author: ```suggestion
He has co-founded transformational elements of the project, including the [KEP][enhancements] (Kubernetes Enhancements Proposal) process, the [Release Engineering][releng] subproject, and Working Group Naming. He has previously served as a chair of both SIG PM and SIG Azure.
```

another multiple chairs instance

Across the wider Cloud Native Computing Foundation ([CNCF][cncf]) ecosystem, he is a former [TAG Contributor Strategy][contrib-strat] Chair, [Dex][dex] maintainer, and Program Chair for [KubeCon / CloudNativeCon][kubecon], the cloud native community's flagship conference.

> Stephen is a [prolific contributor][devstats] to CNCF projects, amongst the top 100 (as of writing) code/content committers since inception of the foundation.

Reviewer: ```suggestion
Stephen is a [prolific contributor][devstats] to CNCF projects, and is amongst the top 100 (as of writing) code/content committers since the inception of the foundation.
```
